daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -63,6 +63,8 @@ When deploying Dapr in a production-ready configuration, it's recommended to dep
 
 HA mode can be enabled with both the [Dapr CLI]({{< ref "kubernetes-deploy.md#install-in-highly-available-mode" >}}) and with [Helm charts]({{< ref "kubernetes-deploy.md#add-and-install-dapr-helm-chart" >}}).
 
+Enabling HA mode in an existing Dapr deployment requires additional steps. Please refer to [this paragraph]({{< ref "#enabling-high-availability-in-an-existing-dapr-deployment" >}}) for more details.
```

Comment:
Looking at the previous paragraph, should people go to the CLI/HELM links above or to the new section in this page? are they referring to different scenarios? It seems like the first one is about installing Dapr with HA and your additional address enabling it on an existing installation. If that is the case please edit the previous sentence too (right now we are sending people to multiple places to achieve HA)